### PR TITLE
Adding descriptions of content that doesn't belong in a code of conduct

### PR DESCRIPTION
**Pull request by aeschright**
I'm trying to separate the apolitical statements from religious protections. How does this read?

### CODE REVIEW

File: resources/code_of_conduct.md [hunk 1]
```diff
@@ -54,6 +54,14 @@ This needs to be specific and appropriate for all of the contexts where your pol
 
 If there's anything else you expect people to do, more details you can provide about your process, or outside resources they can access, include them as well.
 
+### What not to include
+
+Some items are better to leave out of your code of conduct's protected groups. Making political parties or beliefs such a category puts people who want to harm others on the same ground as those who are abused. For example, in the US, registering as a Democrat has a very different impact from joining the neo-Nazi Traditionalist Worker Party. In general, beliefs and behaviors should be under the *unacceptable* or *expected behaviors* sections. This includes which text editor you use, lifestyle choices, and other activities.
+
```
Comment (gabelula) [inline resources/code_of_conduct.md]: "In general, beliefs and behaviors should be under the *unacceptable* or *expected behaviors* sections. This includes which text editor you use, lifestyle choices, and other activities." seems not clear to me (or I do not understand what you mean here). What the text editor has to do with this? 

Mentioning a "guide to communication" as a place that can detail more information on how to help people navigate through cultural differences would be good.

File: resources/code_of_conduct.md [hunk 2]
```diff
@@ -54,6 +54,14 @@ This needs to be specific and appropriate for all of the contexts where your pol
...
+Religion has a special status, both legally and socially. It stays in the main category of protection because of specific histories of harm and discrimination, and because it's something you "are" as much as something you do. On the other hand, proselytizing without an invitation is a *behavior*, regardless of the beliefs that contribute to it.
+
+Also refrain from statements that are ambiguous or rely on specific cultural knowledge. "Be excellent to each other" may have all sorts of different understandings. Be careful about anything that may rely on a participant understanding the intent of the organizers — if it's important, make it explicit.
```
Comment (gabelula) [inline resources/code_of_conduct.md]: +1

File: resources/code_of_conduct.md (hunk 1 above)
Comment (aeschright) [reply resources/code_of_conduct.md]: Do you have a favorite guide of that sort?

I'l clarify the lifestyle choices text. Here's an example of what I'm referring to:
`Everyone is welcome at HOPE events, regardless of race, class, gender identity or expression, age, ethnicity, religion, political beliefs, disability, sexual orientation, personal appearance, or education level, text editor choice, and other aspects of who we are.`

**Comment by aeschright** [comment]
I'd also like to provide more guidance about anti-fascist statements. For example (from https://github.com/PotassiumES/potassium-es/blob/master/CODE%20OF%20CONDUCT.md#regarding-fascists):
> Regarding Fascists

> If you are are obsessed with community decline, humiliation, victimhood, unity, or purity then you are in violation of this code of conduct.

> If you propose the abandonment of democratic liberties for redemptive violence, internal cleansing, or expansion without ethical or legal restraint then you are in violation of this code of conduct.

File: resources/code_of_conduct.md (hunk 1 above)
Comment (gabelula) [reply resources/code_of_conduct.md]: https://www.internetfreedomfestival.org/wiki/index.php/Guide_to_Communication

**Pull request merged**